composer.json

```diff
@@ -47,7 +47,7 @@
         "laminas/laminas-http": "^2.6.0",
         "laminas/laminas-i18n": "^2.7.3",
         "laminas/laminas-json": "^3.2.0",
-        "laminas/laminas-mail": "^2.9.0",
+        "laminas/laminas-mail": "^2.11.0",
```

Comment:
Hi @fredden,
Could you update laminas/laminas-mail in the Magento framework to fix static test failure:
https://github.com/magento/magento2/blob/2cbb4e6557139340cbeb5c0773fb078f41cfcc6e/lib/internal/Magento/Framework/composer.json#L31
Thank you!